apps/admin-gui-e2e/src/integration/vo-management-perun_admin.spec.js

```diff
@@ -25,238 +28,245 @@ context('Actions', () => {
     cy.intercept('**/vosManager/createVo/**').as('createVo')
       .intercept('**/vosManager/getVos').as('getVos')
       .get('[data-cy=new-vo-button]')
-        .click()
+      .click()
       .wait('@getVos')
       .get('[data-cy=vo-name-input]')
-        .type('test-e2e-vo')
+      .type('test-e2e-vo')
       .get('[data-cy=vo-shortname-input]')
-        .type('test-e2e-vo')
+      .type('test-e2e-vo')
       .get('[data-cy=create-vo-button]')
-        .click()
+      .click()
       .wait('@createVo')
       .wait('@getVos')
       // assert that the vo was created
       .get('[data-cy=vo-name-link]')
-        .contains('test-e2e-vo')
-        .should('exist')
+      .contains('test-e2e-vo')
+      .should('exist')
   });
 
   it('test add attribute', () => {
 
     cy.intercept('**/attributesManager/setAttributes/vo').as('setAttributes')
       .get(`[data-cy=${dbVoName}]`)
-        .click()
+      .click()
       .get('[data-cy=attributes]')
-        .click()
+      .click()
       .get('[data-cy=add-attributes]')
-        .click()
+      .click()
       // get attribute Notification default language
-      .get('[data-cy=notification-default-language-value]')
-        .type('en')
+      .get(`[data-cy=${addedAttribute}-value]`)
+      .type('en')
       .get('[data-cy=save-selected-attributes]')
-        .click()
-      .wait('@setAttributes')
+      .click()
+      .intercept('**/attributesManager/getAttributes/vo**').as('getAttributes')
+      .wait('@getAttributes')
       // assert that attribute exists
-      .get('[data-cy=notification-default-language-value]')
-        .should('exist')
+      .get(`[data-cy=${addedAttribute}-value]`)
+      .should('exist')
   });
 
   it('test delete attribute', () => {
 
-    cy.intercept('**/attributesManager/removeAttributes/**').as('removeAttributes')
+    cy.intercept('**/attributesManager/getAttributes/**').as('getAttributes')
       .get(`[data-cy=${dbVoName}]`)
-        .click()
+      .click()
       .get('[data-cy=attributes]')
-        .click()
+      .click()
       // get attribute Link to AUP
-      .get('[data-cy=link-to-aup-checkbox]')
-        .click()
+      .get(`[data-cy=${deteleAttribute}-checkbox]`)
+      .click()
       .get('[data-cy=remove-attributes]')
-        .click()
+      .click()
       .get('[data-cy=delete-attributes]')
-        .click()
-      .wait('@removeAttributes')
+      .click()
+      .wait('@getAttributes')
```

Comment:
This test is wrong because this wait is waiting for the first call of the getAttributes - but that already happened when the page first loaded before the remove of the attribute
just add wait to getAttributes before clicking on the checkbox